auth/vitest.config.ts

```diff
@@ -15,5 +15,6 @@ export default defineConfig({
             provider: 'istanbul', // or 'v8'
             reporter: ['text', 'json', 'html'],
         },
+        watch: false
```

Comment:
We should be able to remove the extra `run` argument from the vitest commands if we enforce it here